graylog2-web-interface/src/views/components/aggregationwizard/aggregationElements/AggregationElementType.ts

```diff
@@ -24,6 +24,7 @@ export type AggregationElement = {
   allowCreate: (formValues: WidgetConfigFormValues) => boolean,
   order: number,
   addEmptyElement?: (formValues: WidgetConfigFormValues) => WidgetConfigFormValues,
+  removeElement?: (index: number, formValues) => WidgetConfigFormValues,
```

Comment:
Let's name this e.g. `removeElementSection` otherwise you could think it removes the whole aggregation element and all its sections.